noria-server/src/controller/sql/mir/mod.rs

```diff
@@ -936,12 +975,13 @@ impl SqlToMirConverter {
         parent: MirNodeRef,
     ) -> Vec<MirNodeRef> {
         use dataflow::ops::grouped::aggregate::Aggregation;
+        use dataflow::ops::grouped::filteraggregate::FilterAggregation;
         use dataflow::ops::grouped::extremum::Extremum;
         use nom_sql::FunctionExpression::*;
 
         let mut out_nodes = Vec::new();
 
-        let mknode = |over: &Column, t: GroupedNodeType, distinct: bool| {
+        let mknode = |over: &Column, over_else: Option<Literal>, t: GroupedNodeType, distinct: bool, cond: Option<&ConditionExpression>| {
```

Comment:
I think the compiler _might_ now let you get away with not giving the types for all these closure arguments.

Reply:
As someone not yet very familiar with the codebase, I find it helpful to know what all the types are when I read things like this.